content/en/news/releases/1.6.x/announcing-1.6.5/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Announcing Istio 1.6.5
+linktitle: 1.6.5
+subtitle: Patch Release
+description: Istio 1.6.5 patch release.
+publishdate: 2020-07-08
+release: 1.6.5
+aliases:
+    - /news/announcing-1.6.5
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.5 and Istio 1.6.4.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** support to sort the pods by `creationTimestamp` during mixer lookup and return the latest pod.
+- **Improved** Support to control sidecar injection based on revision at a per-pod level ([Issue 24801](https://github.com/istio/istio/issues/24801)
+- **Improved** `istioctl validate` will disallow unknown field from spec ([Issue 24860](https://github.com/istio/istio/issues/24860))
+- **Fixed** changed `stsPort` to `sts_port` in envoy bootstrap file. 
+- **Fixed** keep existing wasm state schema since it can be referenced by state objects later.
+- **Improved** added targetUri in stackdriver_grpc_service.
+- **Improved** updated wasm state to log for Access Log Service.
+- **Updated** Raised default protocol detection timeout from 100 Milliseconds to 5 seconds ([Issue 24379](https://github.com/istio/istio/issues/24379))
+- **Updated** removed UDP port 53 from Istiod service.
```

Comment:
**Removed** UDP port from 53 from Istiod service.

Reply:
make all using lower case now to align previous release notes